app/index.html

```diff
@@ -234,7 +209,7 @@ <h4>First Annual Cornell Fencing Association Dinner</h4>
                    <p>
                    <h5>Cornell Fencing Spring Banquet</h5>
                    <p>
-                       The annual Cornell Fencing Spring banquet has been scheduled for May 4, 2019. We do not have a registration link set up yet, but please save the date on your calendar. More info will be coming soon.
+                       The annual Cornell Fencing Spring banquet has been scheduled for May 4, 2019. We will be honoring Olympian Dick Pew. Click <a href="https://cornelluniversity.imodules.com/s/1717/alumni/event.aspx?sid=1717&gid=2&pgid=17180&content_id=15971">here</a> to register and join other alumni and friends for a wonderful weekend in Ithaca.

```

Comment:
`Click <a>here</a>` is an anti-pattern because "here" is vague. Better options include:

> [Register here]() to spend...
> [Register with Cornell Athletics]() to spend...

etc